app/services/publishing_api_payload.rb

```diff
@@ -34,6 +34,15 @@ def payload
 
 private
 
+  def links
+    associations = document.associations
+    if associations.include?("primary_publishing_organisation") && !associations.include?("original_publishing_organisation")
```

Comment:
It might be nice to extract this out into a method to make it a bit easier to read.